src/sentry/api/endpoints/organization_users.py

```diff
@@ -12,11 +12,7 @@ class OrganizationUsersEndpoint(OrganizationEndpoint):
     def get(self, request, organization):
         projects = self.get_projects(request, organization)
         qs = (
-            OrganizationMember.objects.filter(
-                user__is_active=True,
-                organization=organization,
-                teams__projectteam__project__in=projects,
-            )
+            OrganizationMember.objects.filter(user__is_active=True, organization=organization)
```

Comment:
We typically restrict these lists to things that the user is directly a member of, rather than has access to. Usually this is to restrict the amount of data we fetch in large orgs, and to help reduce clutter in the ui. I think this is more of a question for frontend, since they have a better idea of where `MemberListStore` is used. @billyvg @markstory do you have any thoughts here?

Reply:
I don't think it should be a problem, but can we run a query to check the amount of users that this applies to?